Makefile

```diff
@@ -19,8 +19,8 @@ ifndef CGO_LDFLAGS
     export CGO_LDFLAGS := $(LDFLAGS)
 endif
 
-GO_LDFLAGS := -X github.com/cli/cli/command.Version=$(GH_VERSION) $(GO_LDFLAGS)
-GO_LDFLAGS := -X github.com/cli/cli/command.BuildDate=$(BUILD_DATE) $(GO_LDFLAGS)
+GO_LDFLAGS := -X github.com/cli/cli/internal/build.Version=$(GH_VERSION) $(GO_LDFLAGS)
+GO_LDFLAGS := -X github.com/cli/cli/internal/build.Date=$(BUILD_DATE) $(GO_LDFLAGS)
```

Comment:
Forgot to add in my previous review: since the names of `github.com/cli/cli/command.*` variables were technically part of an external interface for this package (as a build script could set them through ldflags), we should note that this change breaks such build scripts. I'm not sure if anyone was changing these variables manually, but it's conceivable that someone studied our Makefile and recreated parts of it for their build script when packaging up GitHub CLI for their system.

I'm fine with taking a risk to maybe break this. We should note this in our changelog, though. The official external interface to set the version and build date should be though our makefile:

```sh
GH_VERSION=v1.2.3 BUILD_DATE="..." make bin/gh
```